<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorGUI</class>
 <widget class="QMainWindow" name="CalculatorGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="display"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <!-- Erste Reihe -->
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonClear">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonPower">
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonSqrt">
        <property name="text">
         <string>√</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="buttonDivide">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>

      <!-- Zweite Reihe -->
      <item row="1" column="0">
       <widget class="QPushButton" name="button7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="button8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="button9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="buttonMultiply">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>

      <!-- Dritte Reihe -->
      <item row="2" column="0">
       <widget class="QPushButton" name="button4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="button5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="button6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="buttonSubtract">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>

      <!-- Vierte Reihe -->
      <item row="3" column="0">
       <widget class="QPushButton" name="button1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="button2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="button3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>

      <!-- Fünfte Reihe -->
      <item row="4" column="0">
       <widget class="QPushButton" name="buttonSin">
        <property name="text">
         <string>sin</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="button0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="buttonDecimal">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="buttonEqual">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <!-- Theme button will be added here programmatically. -->
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
</ui>